q2_longitudinal/plugin_setup.py

```diff
@@ -72,10 +73,13 @@
         'parametric': ('Perform parametric (ANOVA and t-tests) or non-'
                        'parametric (Kruskal-Wallis, Wilcoxon, and Mann-'
                        'Whitney U tests) statistical tests.'),
-        'drop_duplicates': (
-            'If True, will discard all subject IDs with duplicate samples '
-            'at either state_1 or state_2. If False, will instead '
-            'choose one representative at random from among duplicates.')
+        'replicate_handling': (
+            'Choose how replicate samples are handled. If replicates are '
+            'detected, "error" causes method to fail; "drop" will discard all '
+            'subject IDs with replicate samples at either state_1 or state_2; '
+            '"random" chooses one representative at random from among '
+            'replicates; "mean" and "median" compute average values across '
```

Comment:
`mean` and `median` aren't implemented, right? Can you remove those from this description?